litex/boards/platforms/kc705.py

```diff
@@ -496,7 +496,8 @@ def create_programmer(self):
         elif self.programmer == "impact":
             return iMPACT()
         else:
-            raise ValueError("{} programmer is not supported".format(programmer))
+            raise ValueError("{} programmer is not supported"
+                             .format(self.programmer))
```

Comment:
The normal way to wrap this would be;
```python
    raise ValueError("{} programmer is not supported".format(
        self.programmer))
```
or
```python
    raise ValueError(
        "{} programmer is not supported".format(self.programmer))
```